### ISSUE TITLE
Arc warden spark wraith bug

### ISSUE THREAD

**Issue by RekovniBugreport** [ValveSoftware/Dota2-Gameplay#20899]
### Ability name

Spark wraith

### Description

Currently the spark wraith version that has activation delay and increased damage also incorrectly has the bonus of the other version which is the increased spark wraith slow duration.

At the same time the spark wraith version that has increased wraith movement speed does NOT have increased spark wraith slow duration.

### Example Match ID (and possibly Timestamp)

You can test this in demo mode

**Comment by hperera1** [comment]
True. Maybe the tooltips are just wrong at this point and it's intended?

**Comment by RekovniBugreport** [comment]
> True. Maybe the tooltips are just wrong at this point and it's intended?

Who knows? So many outdated tooltips and bugs in the game at this point you don't know anymore what's intended and what's not. 

There are probably thousands of people who would fix these tooltips for FREE but valve still can't be bothered somehow.
